# ------------------------------------------------------
# DESeq Script
# 18 Mar 2020
# ------------------------------------------------------
# 

## Libraries
library(DESeq2)
library(tidyr)
library(dplyr)
library(ggplot2)
library(scales)
library(ggpubr)
library(wesanderson)
BiocManager::install("vsn")
BiocManager::install("hexbin")
library(vsn)


## Import the counts matrix
countsTable <- read.table("RS_cds2kb_countsMatrix.txt", header=TRUE, row.names=1)
head(countsTable)
dim(countsTable)
countsTableRound <- round(countsTable) # Need to round because DESeq wants only integers
head(countsTableRound)
day10countstable <- countsTableRound %>%
  select(contains("10"))
dim(day10countstable)

## Import the samples description table - links each sample to factors of the experimental design.
# Need the colClasses otherwise imports "day" as numeric which DESeq doesn't like, coula altneratively change to d0, d5, d10
conds <- read.delim("RS_samples.txt", header=TRUE, stringsAsFactors = TRUE, row.names=1, colClasses=c('factor', 'factor', 'factor', 'factor'))
head(conds)
dim(conds)
conds10<- subset(conds, day=="10")
dim(conds10)
head(conds10)


## Let's see how many reads we have from each sample:
colSums(countsTableRound)
mean(colSums(countsTableRound))
barplot(colSums(countsTableRound), las=3, cex.names=0.5,names.arg = substring(colnames(countsTableRound),1,13))
abline(h=mean(colSums(countsTableRound)), col="blue", lwd =2)

# average number of counts per gene
rowSums(countsTableRound)
mean(rowSums(countsTableRound))
median(rowSums(countsTableRound)) # yowza two orders of magnitude lower than the mean

# average number of counts per gene per sample
apply(countsTableRound,2,mean) # why aren't any of these super duper high if the mean per gene was ~3000?




# -------------------------------------------------------------------------

# create a DESeq object
dds<-DESeqDataSetFromMatrix(countData=day10countstable,colData=conds10,
                            design = ~climate + treatment)
dim(dds)


# Filter out genes with dinky read counts
dds<-dds[rowSums(counts(dds)) > 76] # 1 read per sample on average
dim(dds) # 23887 genes

# dds<-dds[rowSums(counts(dds)) > 760] # 10 reads per sample on average
# dim(dds) # 7884 genes

# Run the DESeq model to test for differential gene expression: 1) estimate size factors (per sample), 2) estimate dispersion (per gene), 3) run negative binomial glm
dds<-DESeq(dds)

# list the results generated by DESeq
resultsNames(dds)

# -------------------------------------------------------------------------


# run the model again but add interaction effect.
dds2<-DESeqDataSetFromMatrix(countData=day10countstable,colData=conds10,
                             design = ~climate + treatment + climate:treatment)
dim(dds2)

# filter out low count genes
dds2<-dds2[rowSums(counts(dds2)) > 76] # 1 read per sample on average
dim(dds2) # 23887 genes

# dds2<-dds2[rowSums(counts(dds2)) > 760] # 10 reads per sample on average
# dim(dds2) # 7884 genes

# Run the DESeq model to test for differential gene expression: 1) estimate size factors (per sample), 2) estimate dispersion (per gene), 3) run negative binomial glm
dds2<-DESeq(dds2)

# list the results generated by DESeq
resultsNames(dds2)



# -------------------------------------------------------------------------
# Order and list and summarize results from specific contrasts
# Here you set your adjusted p-value cutoff, can make summary tables of the number of genes differentially expressed (up- or down-regulated) for each contrast
res <- results(dds, alpha= 0.05)
res <- res[order(res$padj),]
head(res)
summary(res)


# Looking at dry conditions vs control -----------------------------------------------------------------------

res_treatCD<-results(dds,name="treatment_D_vs_C", alpha= 0.05)
res_treatCD <- res_treatCD[order(res_treatCD$padj),]
head(res_treatCD)
summary(res_treatCD)

# looking at hot conditions vs control ------------------------------------
res_treatCH<-results(dds,name="treatment_H_vs_C", alpha=0.05)
res_treatCH<-res_treatCH[order(res_treatCH$padj),]
summary(res_treatCH)

# looking at CW climate vs HD climate -----------------------------
res_clim <- results(dds,name="climate_HD_vs_CW")
res_clim <- res_clim[order(res_clim$padj),]
head(res_clim)
summary(res_clim)




# Data visualization ------------------------------------------------------

# MA plot
library(plotrix)
par(mfrow=c(2,1))
plotMA(res_treatCD,ylim=c(-3,3)) # holy shitting dicks it actually worked
corner.label(label='a',figcorner=T)
plotMA(res_treatCH,ylim=c(-3,3))
corner.label(label='b',figcorner=T)
plotMA(res_clim,ylim=c(-3,3))
corner.label(label='c',figcorner=T)


# PCA
vsd<-vst(dds2,blind=FALSE)
data<-plotPCA(vsd,intgroup=c("climate","treatment"),returnData=TRUE)
percentVar<-round(100*attr(data,"percentVar"))

data$treatment <- factor(data$treatment, levels=c("C","H","D"), labels = c("C","H","D"))
data$day <- factor(data$day, levels=c("0","5","10"), labels = c("0","5","10"))

ggplot(data, aes(PC1, PC2, color=climate, shape=treatment)) +
  geom_point(size=4, alpha=0.85) +
  xlab(paste0("PC1: ",percentVar[1],"% variance")) +
  ylab(paste0("PC2: ",percentVar[2],"% variance")) +
  theme_minimal()

# Counts of specific top genes
d <-plotCounts(dds, gene=c("MA_10257300g0010","MA_10323281g0010","MA_10426407g0030"), intgroup = (c("treatment","climate")), returnData=TRUE)
d



p <-ggplot(d, aes(x=treatment, y=count, color=treatment, shape=climate)) + 
  theme_minimal() + theme(text = element_text(size=20), panel.grid.major=element_line(colour="grey"))
p <- p + geom_point(position=position_jitter(w=0.3,h=0), size=3) +
  scale_x_discrete(limits=c("C","H","D"))
p

# heatmap -----------------------------------------------------------------
library(pheatmap)
topgenes <- head(rownames(res_treatCD),20)
mat <- assay(vsd)[topgenes,]
mat <- mat - rowMeans(mat)
df <- as.data.frame(colData(dds)[,c("treatment","climate")])
pheatmap(mat, annotation_col=df)










